maven/JarAssembler.kt

```diff
@@ -28,17 +28,20 @@ import java.io.BufferedInputStream
 import java.io.BufferedOutputStream
```

Comment:
It's cool that this AAR support is added with minimal code change! However, I think the domains of Java and Android are conceptually different (even though Android uses Java code extensively.) As such, I would propose:

- Rename this file (and class) to `Assembler`. Given that we are in the `maven` package, that's unambiguous. `Assembler` should contain all the logic common to both JAR and AAR.
- Introduce two inner classes of `Assembler`: `Jar` and `Aar`, which encapsulate the JAR and AAR-specific code. (They will, presumably, need a reference to the `Assembler` class in their constructors.)